third_party/nimble/port/include/nrf52/syscfg/syscfg.h

```diff
@@ -2182,7 +2182,7 @@
 #endif
 
 #ifndef MYNEWT_VAL_BLE_SM_IO_CAP
-#define MYNEWT_VAL_BLE_SM_IO_CAP (BLE_HS_IO_NO_INPUT_OUTPUT)
+#define MYNEWT_VAL_BLE_SM_IO_CAP (BLE_HS_IO_DISPLAY_YESNO)
```

Comment:
Last time I tried this I didn't work properly because CC2564 has no BLE 4.2 support. Have you verified pairing works with snowy on the OG app with this?

Reply:
I haven't tested, but good point. I have disabled it for CC2564.